src/sessions/BasisTheorySessions.ts

```diff
@@ -0,0 +1,60 @@
+import type { AxiosTransformer } from 'axios';
+import { createRequestConfig } from '@/common';
+import {
+  dataExtractor,
+  transformProxyResponseCamelCase,
+  transformProxyRequestSnakeCase,
+} from '@/common/utils';
+import { BasisTheoryService } from '@/service';
+import type { BasisTheoryServiceOptions } from '@/service';
+import type { RequestOptions } from '@/types/sdk';
+import type {
+  CreateSessionResponse,
+  AuthorizeSessionRequest,
+  Sessions,
+} from '@/types/sdk/services/sessions';
+
+export class BasisTheorySessions
+  extends BasisTheoryService
+  implements Sessions {
+  public constructor(options: BasisTheoryServiceOptions) {
+    const _options = {
+      ...options,
+    };
+
+    // eslint-disable-next-line unicorn/prefer-spread
+    _options.transformRequest = ([] as AxiosTransformer[]).concat(
+      transformProxyRequestSnakeCase,
+      options.transformRequest || []
+    );
+
+    // eslint-disable-next-line unicorn/prefer-spread
+    _options.transformResponse = ([] as AxiosTransformer[]).concat(
+      transformProxyResponseCamelCase,
+      options.transformResponse || []
+    );
+
+    super(_options);
+  }
+
+  public createSession(
```

Comment:
do we want to just call it create and authorize? so we can do bt.sessions.authorize instead of bt.sessions.authorizeSession

Reply:
Sounds good, I'll make that change.